plone/portlet/static/static.py

```diff
@@ -17,15 +20,38 @@
 
 logger = logging.getLogger('plone.portlet.static')
 PLONE5 = getFSVersionTuple()[0] >= 5
+USE_AUTOFORM = IAutoExtensibleForm.implementedBy(base.AddForm) or PLONE5
+try:
+    import plone.app.widgets  # noqa
+    USE_PAE = True
```

Comment:
@petri While this indeed a typo, you are commenting on code from a reverted commit within this pull request; please review the final changes.